docs/current_docs/ci/quickstart/snippets/build/php/src/HelloDagger.php

```diff
@@ -17,7 +18,10 @@ class HelloDagger
 {
     #[DaggerFunction]
     #[Doc('Build the application container')]
-    public function build(Directory $source): Container
+    public function build(
+      #[DefaultPath('/')]
+      Directory $source,
+    ): Container
     {
```

Comment:
```suggestion
    ): Container {
```

[When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together](https://www.php-fig.org/per/coding-style/#45-method-and-function-parameters)